plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/app/Application.js

```diff
@@ -229,9 +229,16 @@ Ext.define('NX.app.Application', {
    * Initialize Ex.Direct remote providers.
    */
   initDirect: function () {
-    var me = this;
+    var me = this,
+        remotingProvider;
+
+    remotingProvider = Ext.direct.Manager.addProvider(NX.direct.api.REMOTING_API);
+
+    // disable retry
+    remotingProvider.maxRetries = 0;
```

Comment:
Disabling retry, as otherwise in timeout, remoting provider will attempt the request again.  So in the case of support-zip (before this change) if it took longer than 30 seconds, then 2 support-zips will be created.